homeassistant/components/scrape/sensor.py

```diff
@@ -65,6 +67,9 @@
         vol.Optional(CONF_AUTHENTICATION): vol.In(
             [HTTP_BASIC_AUTHENTICATION, HTTP_DIGEST_AUTHENTICATION]
         ),
+        vol.Optional(
+            CONF_CHARACTER_ENCODING, default=DEFAULT_CHARACTER_ENCODING
+        ): cv.string,
```

Comment:
Platform yaml has been deprecated so should not add new options here

Reply:
Should be added in `__init__` instead as it's configured from integration key.
Also probably it should be added to config flow

Reply:
Ok. Changes made. I tested the config flow for the scrape sensor and it seems to work fine.

Thanks again for your help!